### DISCUSSION TITLE
Sorted list of forms

### DISCUSSION THREAD

**Discussion by wolispace** [solspace/craft-freeform#1492] [Feature Requests]
I notice my forms are no longer shown in alphabetical order.. they appear random but Im guessing its based on created date.

Im in the process of migrating to Freeform5 and trying to compare forms from 4 to 5 is time consuming in that I need to find the form in the list (of about 20 forms) and they are displayed as cards not a list.. so we don't see their full name and they are random on the screen.

Can we have an option to sort the list of forms alphabetically as well as whatever the sort order is you current apply?

**Comment by kjmartens** [comment]
Hi @wolispace,

We're actually working on this right now. We hope to have a new version with this feature (toggling between list and card view, sorting alphabetically, etc) available within 1-2 weeks. 🙂

**Comment by kjmartens** [comment]
This has now been added in Freeform 5.6+ 🙂

**Comment by wolispace** [comment]
The new list of forms is excellent! 

We can right-click and open in a new tab, the columns of info are useful and the graph is a nice touch. 

Great job guys!

**Reply by kjmartens** [reply]
Glad you like it. 🙂
